test/specs/scale.time.tests.js

```diff
@@ -297,8 +297,8 @@ describe('Time scale tests', function() {
 		var xScale = chart.scales.xScale0;
 
 		// Counts down because the lines are drawn top to bottom
-		expect(xScale.ticks[0].value).toBe('Jan 2');
-		expect(xScale.ticks[1].value).toBe('May 8');
+		expect(xScale.ticks[0].value).toEqualOneOf(['Jan 1', 'Jan 2']);
```

Comment:
Strange. Now it's passing for me the way you suggested and failing with the timestamps version. I swear I tried that earlier. Anyway, glad it's passing now. Thanks for the help!